packages/react-examples/src/react-components/Migrations/utils.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Source } from '@storybook/addon-docs/blocks';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const codeComparisonUseStyles = makeStyles({
+  root: {
+    display: 'flex',
+    justifyContent: 'space-between',
+
+    '> *': {
+      width: '100%',
+    },
+
+    '> :not(:last-child)': {
+      marginRight: '5%',
+    },
+  },
+});
+
+export const CodeComparison = (props: { children: React.ReactElement[] }) => {
+  const { children } = props;
+  const classes = codeComparisonUseStyles();
+
+  return <div className={classes.root}>{children}</div>;
+};
+
+export const CodeLanguages: { [key: string]: string } = {
+  html: 'HTML',
+  css: 'CSS',
+  js: 'JavaScript',
+  jsx: 'React',
+};
+export const CodeExample = (props: { title?: string; children: React.ReactElement }) => {
+  const { title, children } = props;
+  // Access the raw values from the markdown source code block
+  const rawValue: string = children?.props?.children?.props?.children;
+  const isMarkdownCodeBlock = rawValue !== undefined;
+
+  if (isMarkdownCodeBlock) {
+    // JSX source blocks are passed without the backticks
+    const language = rawValue.indexOf('```') == 0 ? rawValue.substring(3, rawValue.indexOf('\n')) : 'jsx';
+    const code = rawValue
+      .replace(`\`\`\`${language}`, '')
+      .replace('```', '')
+      .trim();
+
+    return (
+      <div>
+        <h3>{title ?? CodeLanguages[language]}</h3>
```

Comment:
I think  `??` will only work for `null`

```suggestion
        <h3>{title && CodeLanguages[language]}</h3>
```

Reply:
`??` will fallback on `null` and `undefined` values but not on _falsy_ values.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

Reply:
ah ok I stand corrected